_posts/Friends of 4CSCC

```diff
@@ -0,0 +1,30 @@
+---
+---
```

Comment:
We should add a title with proper capitalization so that the post link will look correct.